plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -770,6 +783,50 @@ export class EventsProcessor {
         }
     }
 
+    private tryStoreSessionRecordingToObjectStorage(
+        timestamp: DateTime,
+        session_id: string,
+        snapshot_data: Record<any, any>,
+        team_id: number
+    ): Record<any, any> {
+        // As we don't want to store the session recording payload in ClickHouse,
+        // let's intercept the event, parse the metadata and store the data in
+        // our object storage system.
+
+        if (!this.objectStorage.isEnabled) {
+            return snapshot_data
+        }
+
+        if (team_id !== 2) {
```

Comment:
I gather this needs to be replaced with `ENABLED_SESSION_RECORDINGS_OBJECT_STORAGE_TEAM_IDS` or something less long!